### DISCUSSION TITLE
Custom Sass in the Theme editor doesn't save

### DISCUSSION THREAD

**Discussion by X3Technology** [vanjarosoftware/Vanjaro.Platform#1701] [General]
I edited theme.editor.json and set DeveloperMode to true.  This enables the Update button in the Theme Builder UX.

I then tried to enter some custom CSS under the Site section:
![custom_sass](https://user-images.githubusercontent.com/1585717/131201047-dcb491f8-d827-4efa-b14e-fe4b727a2e46.PNG)

After I click Update, and then go back, my new Sass value was not saved.

**Comment by X3Technology** [comment]
Correction.  Upon looking in theme.editor.json, I see the new Sass value.  However, all of the Controls have been wiped out!

![custom_sass2](https://user-images.githubusercontent.com/1585717/131201188-7e2a0d8e-d1ca-4f6f-884b-c662833ec26c.PNG)

**Comment by meetmandeep** [comment]
Developer Mode is meant to make changes to the Basic Theme; this should only be used if you're intending to contribute these changes as a Pull Request. Otherwise, a new custom theme should be created. It's exciting to see the community uncover features that are not yet officially documented. I would recommend to bring these questions to our dev series webinars starting early next month.

**Comment by X3Technology** [comment]
Will do.  And I did make a copy of the Basic folder before editing the theme.editor.json file.  But I'm pretty sure this is still a bug.  I doubt it was intentional to wipe out all the control if any custom Sass has been entered.

**Comment by meetmandeep** [comment]
>  But I'm pretty sure this is still a bug. 

Yes. Dev team will tag it accordingly after they're able to confirm it.

**Comment by meetmandeep** [comment]
> And I did make a copy of the Basic folder before editing the theme.editor.json file. 

I missed this. Did you try the developer mode on a copy of your theme? If so, it wouldn't work as explained earlier that copying a theme folder is not sufficient to create a new theme.

**Comment by X3Technology** [comment]
It sure does work.  I've got two custom themes going now.
![Themes](https://user-images.githubusercontent.com/1585717/131494048-323340bc-bdc1-40aa-8a5a-3f6fff258bca.PNG)

**Comment by mksmanish79** [comment]
We are not able to duplicate the issue. You may be getting the issue as the steps you followed to copy a theme is not correct.

**Comment by X3Technology** [comment]
I just tried it again and I was unable to duplicate it as well.  I will keep playing with it to see if I can reproduce the steps that resulted in the issue.
